src/components/SingleArticle.jsx

```diff
@@ -14,6 +14,7 @@ export default function SingleArticle({ isLoading, setIsLoading }) {
     });
   }, [article_id]);
 
+  
```

Comment:
It looks like you might have forgot to push your changes to this branch when you were implementing this code! Try to remember going forward, I've looked at your main branch 🌳 
- The styling of the comments looks great! On desktop it looks fantastic though does go a bit funny on smaller screens, so you may want to circle back to this at some point 📱 
- Currently, everything breaks when you go to an article with no comments (like article 37). This may be potentially be a backend problem, but try and work with the backend functionality you have to indicate to the user when an article has no comments
- Fantastic that you're implementing loading patterns throughout the app! However you currently have one loading state for the whole app (initialised in App.jsx) which you're passing down throughout the app. Each instance of something loading should have it's own loading state initialised in the component it's affecting. This would mean that the comments could still be loading while the single article has already loaded.
-It's looking great so far though Anna well done! 🙌